src/instructlab/model/chat.py

```diff
@@ -373,6 +375,42 @@ def _handle_list_contexts(self, _):
         self._sys_print(Markdown(f"**Available contexts:**\n\n{context_list}"))
         raise KeyboardInterrupt
 
+    def _handle_set_custom_context(self, content):
+        content = content.rstrip()
+        parts = content.split(" ", 2)
+        if len(parts) < 3:
+            self._sys_print(
+                Markdown(
+                    "**ERROR**: Please use `/sc context_name custom_context_text`. "
+                    "Note: Use underscores (`_`) to join multiple words in `context_name`."
+                )
+            )
+            raise KeyboardInterrupt
+
+        context_name = parts[1]
+        custom_context_text = parts[2]
+
+        # Prevent overriding default contexts
+        if context_name in CONTEXTS:
```

Comment:
Don't you now forbid overriding an existing context too? (The message suggests it's not your intent; and I think we should be able to override a custom context.)